### PR TITLE
fix link to Deno Standard APIs

### PR DESCRIPTION
**Pull request by guigui64**
Removed redundant "v"

### CODE REVIEW

File: references.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This chapter contains links to reference manuals and sheets, including:
 
-- [Deno Standard APIs](https://deno.land/api@v$CLI_VERSION)
+- [Deno Standard APIs](https://deno.land/api@$CLI_VERSION)
```
Comment (dsherret) [inline references.md:5]: This seems to break the link?

Comment (dsherret) [reply references.md:5]: Nevermind. I see `v` is in $CLI_VERSION

**Review by dsherret — APPROVED** [review]
LGTM. Thanks!

**Comment by bartlomieju** [comment]
Hey @guigui64 thanks for the PR, please sign the CLA so we can land it

**Comment by kt3k** [comment]
Thanks for the PR, but this has been already fixed in https://github.com/denoland/manual/pull/439

**Pull request closed without merging**